yb-voyager/src/migassessment/sizing.go

```diff
@@ -639,32 +641,33 @@ func getExperimentFile(assessmentMetadataDir string) (string, error) {
 		}
 	}
 	if !fetchedFromRemote {
-		err := os.WriteFile(assessmentMetadataDir+experimentDataFileName, experimentData20240, 0644)
+		err := os.WriteFile(filepath.Join(AssessmentMetadataDir, EXPERIMENT_DATA_FILENAME), experimentData20240, 0644)
 		if err != nil {
 			return "", fmt.Errorf("failed to write experiment data file: %w", err)
 		}
 	}
-	return assessmentMetadataDir + experimentDataFileName, nil
+
+	return filepath.Join(AssessmentMetadataDir, EXPERIMENT_DATA_FILENAME), nil
 }
 
-func checkAndDownloadFileExistsOnRemoteRepo(assessmentMetadataDir string) (bool, error) {
+func checkAndDownloadFileExistsOnRemoteRepo() (bool, error) {
 	// check if the file exists on remote github repository using the raw link
-	remotePath := GITHUB_RAW_LINK + experimentDataFileName
+	remotePath := GITHUB_RAW_LINK + EXPERIMENT_DATA_FILENAME
```

Comment:
Please check, I see since both of the vars doesn't have `/` at end or start, you might need to use filepath.Join() here or concat them like this:
```
remotePath := GITHUB_RAW_LINK + "/" + EXPERIMENT_DATA_FILENAME
```

Reply:
done. Concatenated with 
```
remotePath := GITHUB_RAW_LINK + "/" + EXPERIMENT_DATA_FILENAME
```